.github/workflows/test-full-sync.yml

```diff
@@ -212,6 +217,27 @@ jobs:
         env:
           ZEBRA_CONTAINER: ${{ steps.get-container-name.outputs.zebra_container }}
 
+      - name: Get sync height from logs
+        id: get-sync-height
+        run: |
+          INSTANCE_ID=$(gcloud compute instances describe full-sync-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} --zone ${{ env.ZONE }} --format='value(id)')
+          SYNC_HEIGHT=$(gcloud logging read '(resource.type="gce_instance" AND resource.labels.instance_id="${INSTANCE_ID}")' --format='value(jsonPayload.MESSAGE)' --limit 500 | grep 'finished initial sync to chain tip' |  grep -o -E 'Height\(\d+\)' | grep -o -E '\d+')
```

Comment:
Question: There is a 500 line limit here. Does this read the earliest lines first, or the latest lines first?

I'm asking for two reasons:
- "finished initial sync to chain tip" comes very close to the end
- "finished initial sync to chain tip" can be logged multiple times

Reply:
It reads the newest first by default: 
<img width="476" alt="image" src="https://user-images.githubusercontent.com/552961/160727833-bfa1433b-94fe-4f86-b924-264f424de9d8.png">
https://cloud.google.com/sdk/gcloud/reference/logging/read#--limit

Reply:
I can make this explicit with `--order=desc`

Reply:
> I can make this explicit with `--order=desc`

Thanks, I think that would be helpful.